### PR TITLE
Add getRecords example

### PR DESCRIPTION
**Pull request by gravitypriest**
### What does this PR do?

Demonstrates the usage of the getRecords wire adapter.

## The PR fulfills these requirements:

[x] Tests for the proposed changes have been added/updated.
[x] Code linting and formatting was performed.

### CODE REVIEW

**Comment by gravitypriest** [comment]
Could I get a review from one of the maintainers?

File: force-app/main/default/lwc/wireGetRecord/wireGetRecord.js
Comment (satyasekharcvb) [inline force-app/main/default/lwc/wireGetRecord/wireGetRecord.js:8]: Hard-coded value has to be removed. Instead, you can use @salesforce/user/Id. Something like this--
import Id from '@salesforce/user/Id';
......
......
userId = Id;
    @wire(getRecord, { 
        recordId: '$userId', 
        fields: [NAME_FIELD], 
        optionalFields: [EMAIL_FIELD]  
    }) record;
.....
.......

File: force-app/main/default/lwc/wireGetRecord/wireGetRecords.js
Comment (satyasekharcvb) [inline force-app/main/default/lwc/wireGetRecord/wireGetRecords.js:1]: Is this supposed to be a separate component? Please create the component in its own folder.

Comment (satyasekharcvb) [inline force-app/main/default/lwc/wireGetRecord/wireGetRecords.js:1]: api is not used anywhere in the code.

File: force-app/main/default/lwc/wireGetRecord/wireGetRecords.js
Comment (satyasekharcvb) [inline force-app/main/default/lwc/wireGetRecord/wireGetRecords.js:10]: hard coded ids should be avoided. It will fail as the same ids will not be available in the deployed org.

File: force-app/main/default/lwc/wireGetRecord/__tests__/wireGetRecord.test.js
Comment (satyasekharcvb) [inline force-app/main/default/lwc/wireGetRecord/__tests__/wireGetRecord.test.js:6]: getObjectInfo.json is not found in the data folder

**Review by satyasekharcvb — CHANGES_REQUESTED** [review]
Sorry for the delay in reviewing as I had a problem with my laptop and had to reconfigure a new laptop. I have added a few comments. Thank you!

File: force-app/main/default/lwc/wireGetRecord/wireGetRecords.js
Comment (gravitypriest) [reply force-app/main/default/lwc/wireGetRecord/wireGetRecords.js:10]: How would you suggest specifying two unique IDs here using the approach you suggested for the other file?

Comment (satyasekharcvb) [reply force-app/main/default/lwc/wireGetRecord/wireGetRecords.js:10]: You can wire the Apex method to retrieve user records - https://developer.salesforce.com/docs/component-library/documentation/en/lwc/apex_wire_method

**Comment by gravitypriest** [comment]
@satyasekharcvb I've updated this PR with the requested changes, however I get this issue when running tests:

```
 FAIL  force-app/main/default/lwc/wireGetRecords/__tests__/wireGetRecords.test.js
  ● Test suite failed to run

    TypeError: Invalid adapterId, it must be extensible.



      at node_modules/@lwc/jest-preset/src/setup.js:131:23
          at Array.forEach (<anonymous>)
      at overriddenRegisterDecorators (node_modules/@lwc/jest-preset/src/setup.js:118:23)
      at Object.<anonymous> (force-app/main/default/lwc/wireGetRecords/wireGetRecords.js:82:29)
      at Object.<anonymous> (force-app/main/default/lwc/wireGetRecords/__tests__/wireGetRecords.test.js:2:1)
 ```
 
 It only seems to happen with the getRecords adapter. Any ideas?

**Comment by gravitypriest** [comment]
Can someone review the changes? Thanks!

File: force-app/main/default/lwc/wireGetRecordsDifferentTypes/wireGetRecordsDifferentTypes.js
Comment (satyasekharcvb) [inline force-app/main/default/lwc/wireGetRecordsDifferentTypes/wireGetRecordsDifferentTypes.js:39]: You may remove this line as the contacts property is not used anywhere.

File: force-app/main/default/lwc/wireGetRecord/wireGetRecord.html
Comment (satyasekharcvb) [inline force-app/main/default/lwc/wireGetRecord/wireGetRecord.html:14]: Can you please provide more description, something like 'Get a record's data using @wire'. Also, change the description for other components.

Comment (satyasekharcvb) [reply force-app/main/default/lwc/wireGetRecord/wireGetRecord.html:14]: Hi @gravitypriest, Thank you for updating the PR with the new commits. I have left a couple of minor comments. Also, Can you please run prettier once and fix the formatting issues?

File: force-app/main/default/lwc/wireGetRecord/wireGetRecord.html
Comment (satyasekharcvb) [inline force-app/main/default/lwc/wireGetRecord/wireGetRecord.html:13]: can you please change the source to "lwc/wireGetRecord"

File: force-app/main/default/lwc/wireGetRecords/wireGetRecords.html
Comment (satyasekharcvb) [inline force-app/main/default/lwc/wireGetRecords/wireGetRecords.html:13]: Can you please change the source to "lwc/wireGetRecords"

File: force-app/main/default/lwc/wireGetRecordsDifferentTypes/wireGetRecordsDifferentTypes.html
Comment (satyasekharcvb) [inline force-app/main/default/lwc/wireGetRecordsDifferentTypes/wireGetRecordsDifferentTypes.html:13]: can you please change the source to "lwc/wireGetRecordsDifferentTypes"

**Comment by gravitypriest** [comment]
@satyasekharcvb updated!

**Comment by satyasekharcvb** [comment]
Hi @gravitypriest, Thank you for updating. I have approved the changes. Somehow cla-signed tag is missing. Can you please try to sign the cla once again? If you face issues please let us know, I will reach out to the respective team and merge the changes.

**Pull request merged**
